wgpu/src/api/buffer.rs

```diff
@@ -240,8 +240,9 @@ impl Buffer {
     ///
     /// - If `bounds` is outside of the bounds of `self`.
     /// - If `bounds` has a length less than 1.
+    #[track_caller]
     pub fn slice<S: RangeBounds<BufferAddress>>(&self, bounds: S) -> BufferSlice<'_> {
-        let (offset, size) = range_to_offset_size(bounds);
+        let (offset, size) = std::dbg!(range_to_offset_size(bounds, self.size));
```

Comment:
```suggestion
        let (offset, size) = range_to_offset_size(bounds, self.size);
```

dbg slipped in? thought clippy [would catch those](https://rust-lang.github.io/rust-clippy/master/index.html#/dbg_macro) but looks like we have to activate that (: